### ISSUE TITLE
When Moonlight is running, the system cannot receive all input events?

### ISSUE THREAD

**Issue by GeorgeWang12356** [moonlight-stream/moonlight-embedded#871]
Please provide the following info.

**_NVidia Geforce Experience version:_**
**Moonlight Embedded version:**
**Moonlight Embedded source:** _repository/included in distribution/compiled from source/..._
**Moonlight Embedded running on:** _Raspberry Pi/Cubox-i/Hummingboard/Other linux device/..._
**Moonlight Embedded running on distribution:** _Arch Linux/Raspbian/OpenELEC/..._

**Verbose output `-verbose` of Moonlight Embedded:**
Moonlight Embedded 2.5.3
**What is the expected result?**
When Moonlight is running, other applications can accept input events through the input subsystem

**Comment by cgutman** [comment]
That is correct. When running from the console, we use `EVIOCGRAB` to prevent things like Ctrl+C and similar from being passed to the terminal and killing Moonlight.
